test/e2e/mock-e2e.js

```diff
@@ -18,6 +18,7 @@ const {
  * @returns
```

Comment:
update return statement, might also be helpful to add a type for the request in the .on handler so in the future others know how to extend this functionality if needed.